@ndlib/gatsby-theme-marble/src/components/Shared/PartiallyDigitized/index.js

```diff
@@ -0,0 +1,26 @@
+/** @jsx jsx */
+// eslint-disable-next-line no-unused-vars
+import React from 'react'
+import PropTypes from 'prop-types'
+import { jsx } from 'theme-ui'
+const PartiallyDigitized = ({ iiifManifest }) => {
+  console.log(iiifManifest)
```

Comment:
Do you want to take this `console.log` out, or is this still WIP?

Reply:
Remove. This is why we review.